runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -113,17 +113,20 @@ private String buildToolTip(ItemManager itemManager, ItemStack item)
 	{
 		final String name = itemManager.getItemComposition(item.getId()).getName();
 		final int quantity = item.getQuantity();
-		final long price = calculatePrice(itemManager, new ItemStack[]{item});
+		ItemPrice p = itemManager.getItemPrice(itemManager.getUnnotedItemComposition(item.getId()).getId());
```

Comment:
You are getting itemComposition while not being at clientThread, this is potentionally dangerous